### ISSUE TITLE
Landing page GTR buttons

### ISSUE THREAD

**Issue by AlistairNWard** [iobio/genepanel.iobio#260]
I assume the learn more button will ultimately point to the GTR blog post?

I think we should add another button that links out to the GTR webpage.

**Comment by adityaekawade** [comment]
Yes, the learn more button will point to the blog post. I will add another button that will link to the GTR website. Do we also need a similar button for phenolyzer?

**Comment by AlistairNWard** [comment]
Yes, we do. I was going to make the same comments for Phenolyzer when I got there!

**Comment by adityaekawade** [comment]
This is very basic, but what should be the text on these buttons? 
NCBI for Gtr and WGLAB for phenolyzer?

**Comment by AlistairNWard** [comment]
The Phenolyzer button should probably go to the Phenolyzer page rather than the WGLab. The link in the text will go to that page.

Agreed it seems simple, but isn't obvious what should be on the buttons. Maybe "About GTR" and "About Phenolyzer"?
